### DISCUSSION TITLE
organization_packages permission

### DISCUSSION THREAD

**Discussion by thepwagner** [community/community#32195] [Actions] [answered]
According to [Create an installation access token for an app](https://docs.github.com/en/rest/apps/apps#create-an-installation-access-token-for-an-app), `organization_packages` is an acceptable permission.
The same string, `organization_packages`, is referenced as part of application permissions in several [Octokit](https://github.com/search?q=org%3Aoctokit+organization_packages&type=code) libraries.
This seems like a permission that GitHub apps should be able to request.

But when I go to my application's settings, `https://github.com/settings/apps/my-app/permissions` , I don't see how I can request that permissions in my app. I see the `packages` permission, and several `organization_*` permissions like `organization_custom_rules` - but no `organization_packages`.

What am I missing?
I'm using GitHub.com, is it possible this permission is only available on GitHub Enterprise? I wouldn't be surprised if this is a feature flag in the wrong state.

I've tried:
* Using a ~years old app
* Creating a new app
* Using an app owned by an organization that has packages

**Comment by thepwagner** [comment, marked as answer]
GitHub support has confirmed: the documentation is incorrect.

GitHub.com does not support the `organization_packages` permission - but this may be added in the future.

**Comment by krische** [comment]
Disappointing to see that this functionality has still not been added three years later 😢
